src/ex3/helper.js

```diff
@@ -0,0 +1,67 @@
+import axios from 'axios'
```

Comment:
this file could be seperated into different classes with different responsibilities.